src/plugins/vis_type_table/public/table_vis_response_handler.ts

```diff
@@ -30,36 +30,38 @@
 
 import { getFormatService } from './services';
 import { OpenSearchDashboardsDatatable } from '../../expressions/public';
-import { TableVisConfig } from './types';
+import { FormattedColumn, TableVisConfig } from './types';
+import { convertToFormattedData } from './utils/convert_to_formatted_data';
 
 export interface Table {
   columns: OpenSearchDashboardsDatatable['columns'];
   rows: OpenSearchDashboardsDatatable['rows'];
 }
 
+export interface TableContext {
+  rows: OpenSearchDashboardsDatatable['rows'];
+  columns: OpenSearchDashboardsDatatable['columns'];
+  formattedColumns: FormattedColumn[];
+}
```

Comment:
nit
```suggestion
export interface TableContext extends Table {
  formattedColumns: FormattedColumn[];
}
```

I wonder if `FormattedTableContext` would be a little more meaningful of a name?

Reply:
Will change